### PR TITLE
focuses guide title when linking from guide prev/next

### LINKED ISSUE #450: Guides: Previous/Next button to go to main content for VoiceOver users
Following on from #444

Previous/Next button when VoiceOver on Mac is switched on still brings users to the top of the page rather than the main content is intended.

Since one of the reasons for this change was to benefit screenreader users, we should take a look at this and see if we can change something that it works for VoiceOver users too. We could also take a look at this with NVDA or JAWS and see if we observe similar behaviour.

### PR DESCRIPTION
**Pull request by markconroy**
Closes #450

### CODE REVIEW

**Comment by keelanfh** [comment]
This doesn't seem to work for me, not sure if I'm doing something wrong, but I'm seeing it work just the same as before. Does it work for you with VoiceOver?

File: localgov_base.libraries.yml [hunk 1]
```diff
@@ -124,6 +124,10 @@ topic-list-builder:
     theme:
       css/components/topic-list-builder.css: {}
 
+guide:
```
Comment (keelanfh) [inline localgov_base.libraries.yml]: should this be 'guides'? or else change in the attach_library so that it matches?

**Comment by finnlewis** [comment]
Leaving this for @markconroy to review

**Comment by keelanfh** [comment]
Still doesn't work for me - I'm not sure if I'm just not clearing caches properly or something? Can someone else test it?

**Comment by markconroy** [comment]
@keelanfh what's not working exactly? As in, when you click "next" and go to the next guide page, what is set as the focussed item? 

In my testing, it was the title. So if I hit tab I then go to the first link inside that guide item, and if it hit Shift+Tab I go to the last item in the guide nav. Is this not happening for you?

**Comment by keelanfh** [comment]
No... for me when I hit the next button, it's just bringing focus to the "Skip to main content" link... that's why I'm wondering whether I've applied the patch correctly, as I can't see the javascript in there. I've taken the same approach as before, but happy if you want to merge/release and then I can look at it again

**Review by keelanfh — APPROVED** [review]

**Comment by finnlewis** [comment]
@markconroy shall we go ahead and merge and release this?

**Comment by markconroy** [comment]
Yes, I think so.

**Pull request merged**